examples/ci.json

```diff
@@ -1,17 +1,40 @@
 {
-    "atom_features": [
-        0
-    ],
-    "batch_size": 32,
-    "hidden_dim": 10,
-    "num_conv_layers": 5,
-    "learning_rate": 0.02,
-    "radius": 7,
-    "max_num_node_neighbours": 4,
-    "num_epoch": 50,
-    "perc_train": 0.7,
-    "output_dim": 1,
-    "predicted_value_option": 1,
-    "dataset_option": "unit_test",
-    "model_type": "PNN"
-}
+    "Dataset": {
```

Comment:
@streeve 
For now, let's keep it as is, because not all the properties are input, some of them are also output.
We will open another PR in case we come up with a better solution.